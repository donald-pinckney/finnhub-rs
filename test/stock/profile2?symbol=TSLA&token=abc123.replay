{"request":{"url":"https://finnhub.io/api/v1/stock/profile2?symbol=TSLA&token=abc123","method":"GET","body":null,"headers":{}},"response":{"url":"https://finnhub.io/api/v1/stock/profile2?symbol=TSLA&token=abc123","status":200,"headers":{"access-control-allow-credentials":"true","access-control-allow-headers":"Origin","access-control-allow-methods":"POST, GET, OPTIONS, PUT, DELETE","access-control-allow-origin":"*","access-control-expose-headers":"Content-Length","alt-svc":"h3-27=\":443\"; ma=86400, h3-28=\":443\"; ma=86400, h3-29=\":443\"; ma=86400","cf-cache-status":"DYNAMIC","cf-ray":"644c3cf74f1b3fc7-ORD","cf-request-id":"09a3846e8a00003fc73a362000000001","connection":"keep-alive","content-type":"application/json; charset=utf-8","date":"Sat, 24 Apr 2021 03:27:50 GMT","expect-ct":"max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"","nel":"{\"max_age\":604800,\"report_to\":\"cf-nel\"}","report-to":"{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report?s=CJzTdISkBnt8fBCbNKoKy%2Bll0HYuRvDd0w5c1%2BX6Ps14i6jYLl1gU%2B%2BA1wQWh%2F2w%2B1IKWIr4M6TNgIlrd4K%2FjTBIDcXS6PTVBB0tPjchwAYNW2QokqX6\"}],\"max_age\":604800,\"group\":\"cf-nel\"}","server":"cloudflare","set-cookie":"__cfduid=d4fa34043f81174a7c6d5eec996cb7d811619234870; expires=Mon, 24-May-21 03:27:50 GMT; path=/; domain=.finnhub.io; HttpOnly; SameSite=Lax; Secure","transfer-encoding":"chunked","x-ratelimit-limit":"60","x-ratelimit-remaining":"50","x-ratelimit-reset":"1619234930"},"body":"eyJjb3VudHJ5IjoiVVMiLCJjdXJyZW5jeSI6IlVTRCIsImV4Y2hhbmdlIjoiTkFTREFRIE5NUyAtIEdMT0JBTCBNQVJLRVQiLCJmaW5uaHViSW5kdXN0cnkiOiJBdXRvbW9iaWxlcyIsImlwbyI6IjIwMTAtMDYtMjkiLCJsb2dvIjoiaHR0cHM6Ly9maW5uaHViLmlvL2FwaS9sb2dvP3N5bWJvbD1UU0xBIiwibWFya2V0Q2FwaXRhbGl6YXRpb24iOjcxNDI0Ni4yLCJuYW1lIjoiVGVzbGEgSW5jIiwicGhvbmUiOiIxNjUwNjgxNTAwMC4wIiwic2hhcmVPdXRzdGFuZGluZyI6OTU5Ljg1MzUwNCwidGlja2VyIjoiVFNMQSIsIndlYnVybCI6Imh0dHBzOi8vd3d3LnRlc2xhLmNvbS8ifQ=="},"format_version":3}