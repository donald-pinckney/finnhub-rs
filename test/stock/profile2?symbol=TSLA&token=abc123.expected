CompanyProfile {
    country: "US",
    currency: "USD",
    exchange: "NASDAQ NMS - GLOBAL MARKET",
    ipo: "2010-06-29",
    name: "Tesla Inc",
    phone: "16506815000.0",
    shareOutstanding: 959.853504,
    ticker: "TSLA",
    weburl: "https://www.tesla.com/",
    logo: "https://finnhub.io/api/logo?symbol=TSLA",
    finnhubIndustry: "Automobiles",
}
